requirements/py3_common.txt

```diff
@@ -2,3 +2,12 @@
 
 # Used for Hesiod lookups, etc.
 py3dns==3.1.0
+
+# Django extension for static asset pipeline
+django-pipeline==1.6.8
```

Comment:
It is better to remove it from both `py2_common.txt` and `py3_common.txt` and move it to `common.txt`. `py2_common.txt` and `py3_common.txt` should be used for dependencies specific to a python version, while `django-pipeline==1.6.8` runs on both.